ibm/mas_devops/roles/suite_app_install/README.md

```diff
@@ -107,11 +107,10 @@ Optional. Defines what plan will be used in application install.
 Role Variables - Visual Inspection Configuration
 -------------------------------------------------------------------------------
 ### mas_app_settings_visualinspection_storage_class
-Storage class used for user data. This must support ReadWriteOnce
+Optional - Storage class used for user data. This must support ReadWriteOnce
```

Comment:
must be ReadWriteMany

Reply:
ReadWritemany has been updated.

Not adding MVI EFS requirement as it is not required for MAS 8.10.  Customer can select EFS storage class using MVI 8.8.